eclass/tests/systemd.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
Currently only tests/unpacker.sh use this, but fwiw this matters on prefix and wouldn't hurt to use going forward.